### PR TITLE
pull-request-1

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @wedeploy,

Attached is an update for the date in the footer.

Please let me know if you have any questions. Thanks!

### CODE REVIEW

File: node/magnet/src/pages/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ import '../partials/Benefits.soy.js';
 import '../partials/Data.soy.js';
 import '../partials/Email.soy.js';
 import '../partials/Enterprise.soy.js';
-import '../partials/Footer.soy.js';
+import '../partials/Footer.js';
```
Comment (jonnilundy) [inline node/magnet/src/pages/index.js]: I think it might be best for us to transition the footer to be a component instead of a partial.

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
Merged, thanks @jonmak08!

**Pull request closed without merging**